SETUP.md

```diff
@@ -31,20 +31,22 @@ Every user on Nest has resource limits setup to prevent abuse of Nest and to mak
 CPU time and memory are configured through SystemD & CGroups. These default limits are defined in `/etc/systemd/system/user-.slice.d/limit.conf`, and can be overridden for users in `/etc/systemd/system/user-<id>.slice.d/limit.conf`. User overrides override the **entire** configuration - so if you want to increase the max memory for a user, you must specify the high memory and CPU quota in the file as well.
 
 `/etc/systemd/system/user-.slice.d/limit.conf`:
+
 ```
 [Slice]
 MemoryHigh=2G
 MemoryMax=2500M
 CPUQuota=200%
 ```
+
 (see https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html and https://www.kernel.org/doc/Documentation/cgroup-v2.txt for reference)
 
 Note that to apply changes to either the CPU time or memory limits, you must run `systemctl daemon-reload`. You can verify that a limit is applied by checking the appropriate file in `/sys/fs/cgroup/user.slice/user-<id>.slice` (ex. memory.max for MemoryMax).
 
 Disk space limits are configured through `quota`. The following command is run by Nest Bot to set the default disk space limit:
 
 ```sh
-setquota -u <user> 15G 15G 0 0 /
```

Comment:
Why is this modified? the `-u` flag specifies the user (see [setquota(8)](https://linux.die.net/man/8/setquota))

Reply:
fixed. my linter decided it didnt want it so i reverted its changes.